### ISSUE TITLE
Doesn't work with intl 0.16.0. Should always be updated to work with the latest intl version. 

### ISSUE THREAD

**Issue by JKolsps** [jifalops/datetime_picker_formfield#72]

**Comment by klivin** [comment]
https://github.com/jifalops/datetime_picker_formfield/pull/74
fixes this issue, just a version change on the yaml... can we get this into a new release please!

**Comment by valterh4ck3r** [comment]
Same problem.

> intl: ^0.16.0
> datetime_picker_formfield: ^0.4.3



```
Because datetime_picker_formfield 0.4.3 depends on intl ^0.15.8 and no versions of datetime_picker_formfield match >0.4.3 <0.5.0, datetime_picker_formfield ^0.4.3 requires intl ^0.15.8.

So, because project_test depends on both datetime_picker_formfield ^0.4.3 and intl ^0.16.0, version solving failed.
pub get failed (1)
```

**Comment by juliengit2** [comment]
same issue here, any solution ?

**Comment by valterh4ck3r** [comment]
> same issue here, any solution ?

back your intl to 0.15.8 version.

**Comment by sante85** [comment]
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.12.13+hotfix.5, on Linux, locale it_IT.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Android Studio (version 3.5)
[✓] IntelliJ IDEA Ultimate Edition (version 2019.3)


Because datetime_picker_formfield 0.4.3 depends on intl ^0.15.8 and every version of flutter_localizations from sdk depends on intl 0.16.0, datetime_picker_formfield 0.4.3 is incompatible with flutter_localizations from sdk.

So, because plants_market depends on both flutter_localizations any from sdk and datetime_picker_formfield 0.4.3, version solving failed.
pub get failed (1; So, because plants_market depends on both flutter_localizations any from sdk and datetime_picker_formfield 0.4.3, version solving failed.)

**Comment by Reeyyy** [comment]
Temporary Solution:
Try forking the repo and changing the dependency from intl: ^0.15.8 to intl: ^0.16.0.
After, put the folowing in your project pubspec.yaml:
`datetime_picker_formfield:
    git:
      url: https://github.com/youruser/datetime_picker_formfield
      ref: master`

**Comment by agent3bood** [comment]
I used this version to solve the issue
`datetime_picker_formfield: 1.0.0-pre.5`

**Comment by jifalops** [comment]
Sorry for taking so long on this one. This is fixed in v1.0.0.
